content/en/plugins/github-api/install/armory-cd.md

```diff
@@ -0,0 +1,137 @@
+---
+linkTitle: Armory CD
+title: Install the GitHub API Plugin in Armory Continuous Deployment
+weight: 1
+description: >
+  Learn how to install the GitHub API Plugin in Armory CD so that your app developers can trigger a Spinnaker pipeline from a GitHub workflow.
+---
+![Proprietary](/images/proprietary.svg)
+
+## Installation overview
+
+Enabling the GitHub API plugin consists of the following steps:
+
+1. [Meet the prerequisites](#before-you-begin)
+1. [Create and install a GitHub App](#create-and-install-a-github-app)
+1. [Configure the plugin](#configure-the-plugin)
+1. [Install the plugin](#install-the-plugin)
+
+## {{% heading "prereq" %}}
+
+{{< include "plugins/github/install-reqs.md" >}}
+* You are running Armory Continuous Deployment.
+* You manage your instance using the Armory Operator and the `spinnaker-kustomize-patches` [repo](https://github.com/armory/spinnaker-kustomize-patches). 
+
+## Compatibility
+
+{{< include "plugins/github/compat-matrix.md" >}}
+
+## Create and install a GitHub App
+
+{{< include "plugins/github/github-app.md" >}}
+
+## Configure the plugin
+
+Create a `github-api.yml` file with the following contents: 
+
+```yaml
+apiVersion: spinnaker.armory.io/v1alpha2
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  spinnakerConfig:
+    profiles:
+      # Configs in the spinnaker profile get applied to all services
+      spinnaker:
+        github:
+          plugin:
+            accounts: []
+        spinnaker:
+          extensibility:
+            repositories:
+              repository:
+                enabled: true
+                url: https://raw.githubusercontent.com/armory-plugins/pluginRepository/master/repositories.json
+			deck:
```

Comment:
Not sure how to phrase it but it needs to made clear that in the settings-local.js the triggerTypes needs to be appended in the existing settings.
```yaml
			deck:
				settings-local.js: |
            <Existing settings-local.js settings>
						window.spinnakerSettings.triggerTypes=['artifactory','concourse','cron','docker','git','github','helm','jenkins','nexus','pipeline','plugin','pubsub','travis','webhook','wercker']

```